__tests__/check-linked-issue-labels.test.ts

```diff
@@ -0,0 +1,65 @@
+import {describe, expect, it} from '@jest/globals'
+import {checkLinkedIssueLabels} from '../src/check-linked-issue-labels'
+
```

Comment:
is there a use case we should cover in terms of mixed bags? e.g. PR targets more than one issue, where one has the required labels and one or more do not?

Reply:
Right! 171848c